### ISSUE TITLE
Create & maintain a changelog

### ISSUE THREAD

**Issue by mshmsh5000** [DoSomethingArchive/legacy-website#4980]
We need a public, easily-scannable history of changes and releases.

Try the [GitHub Changelog Generator](https://github.com/skywinder/github-changelog-generator) or a similar project for automatic CHANGELOG.md creation.

**Comment by itsjoekent** [comment]
This is something Vision could do, https://github.com/DoSomething/vision -- sergii suggested doing this when I first presented the project https://github.com/DoSomething/vision/issues/2

**Comment by mshmsh5000** [comment]
That's a great idea! [The current PR for this issue](https://github.com/DoSomething/phoenix/pull/4981) uses that generator gem, and I think is sufficient for now.

**Comment by itsjoekent** [comment]
for generation its probably better to use that, the solution I was going to build would just live on the vision site and wouldn't create an actual file.

**Comment by mshmsh5000** [comment]
Closed in favor of #5449.

**Comment by mshmsh5000** [comment]
Summarizing how this can work from the command line:

Install the [changelog generator gem](https://github.com/skywinder/github-changelog-generator).

Run this:

```
#!/bin/sh

# Requires gem github_changelog_generator
# https://github.com/skywinder/github-changelog-generator

github_changelog_generator -t [MY_GITHUB_TOKEN] --no-unreleased --base CHANGELOG.md DoSomething/phoenix
```

Using a GH token here is important, because it allows many more API calls (which you need to make).
